azure-pipeline-templates/build-release.yml

```diff
@@ -44,6 +44,22 @@ steps:
     displayName: "Test Binary built"
     continueOnError: false
 
+  # Build the health-monitor binary  
+  - task: Go@0
+    inputs:
+      command: 'build'
+      arguments: "-o healthmon ./tools/health-monitor/"
```

Comment:
we can instead use ./build.sh script for both build steps so that later if we need any other options its just a change in the script rather than pipelines

Reply:
Done